client/shipping-label/views/index.js

```diff
@@ -25,10 +26,10 @@ const ShippingLabelRootView = ( props ) => {
 		<p className="wcc-metabox-button-container">
 			<GlobalNotices id="notices" notices={ notices.list } />
 			<PrintLabelDialog
-				{ ...( props.shippingLabel ) }
+				{ ...props.shippingLabel }
 				{ ...props }
 			/>
-			{ props.shippingLabel.success ? renderSuccessNotice() : renderPrintLabelFlow() }
+			{ props.shippingLabel.labels.purchased_time ? renderSuccessNotice() : renderPrintLabelFlow() }
```

Comment:
This `successNotice` would be renered *instead of* the "Create Label" button. A couple of weeks from now, instead of a generic "Congratulations, you printed a label" message, the "Refund" button will be there.